cmd/assaf-http/main.go

```diff
@@ -464,34 +428,58 @@ func (rc RabbitMQConf) String() string {
 	return fmt.Sprintf("amqp://%s:%s@%s:%d/", rc.user, rc.pass, rc.host, rc.port)
 }
 
+type Reqs struct {
+	sync.RWMutex
```

Comment:
super simple thread safe map implementation. lifted from somewhere and adapted to our needs. since we are using a very specific type using a the more general sync.Map wouldn't have much advantage